core/sqf/sqenvcom.sh

```diff
@@ -505,23 +505,6 @@ EOF
   APACHE_HIVE_HOME=$HIVE_HOME
   export HIVE_CNF_DIR=$HIVE_HOME/conf
 
```

Comment:
this is used for install_local_hadoop, but when on vanilla env , i also add Hive path(traf_config_setup line 521),if remove this one ,it will also remove the "hive home input" in traf_config_setup